lib/tools/system-calls.js

```diff
@@ -787,13 +787,24 @@ systemCallMethods.reboot = async function (retries = DEFAULT_ADB_REBOOT_RETRIES)
       }
       // this device needs adb to be running as root to stop.
       // so try to restart the daemon
-      log.debug('Device requires adb to be running as root in order to reboot. Restarting daemon');
+      log.debug('Device requires adb to be running as root in order to reboot when running stop command. Restarting daemon');
       await this.root();
       await this.shell(['stop']);
     }
     await B.delay(2000); // let the emu finish stopping;
```

Comment:
can hardcoded delay be replaced with a timer?

Reply:
added the EMU_STOP_TIMEOUT const